oraichain/chain.json

```diff
@@ -34,47 +34,30 @@
   },
   "codebase": {
     "git_repo": "https://github.com/oraichain/orai",
-    "recommended_version": "v0.41.7",
+    "recommended_version": "v0.42.4",
     "compatible_versions": [
-      "v0.41.3",
-      "v0.41.4",
-      "v0.41.6",
-      "v0.41.7"
+      "v0.42.4"
     ],
-    "cosmwasm_version": "0.30.2",
+    "cosmwasm_version": "0.33.0",
     "cosmwasm_enabled": true,
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/oraichain/oraichain-static-files/master/genesis.json"
     },
     "versions": [
       {
-        "name": "v0.40",
-        "recommended_version": "v0.40.3",
+        "name": "v0.42.4",
```

Comment:
Please add v0.42 below v0.40 and v0.41 instead of removing the version history.

Reply:
Dear Sir,
I have added v0.40 and v0.41 as per your comment.